up-l1/zenoh.adoc

```diff
@@ -98,22 +108,24 @@ Different message types **MUST** use different Zenoh API.
 
 === URI Mapping
 
-UUri **MUST** be able to be serialized into link:../basics/uri.adoc#42-micro-uris[micro form].
-Then we can transform the micro-form UUri, which is `u8` array, into string in lowercase hexadecimal representation.
+UUri **MUST** be able to be serialized into link:../basics/uri.adoc#short-uris[short form].
+Then we can transform the short-form UUri into string for Zenoh key.
 
 Zenoh key for uProtocol consists of three parts and they are separated by `/`.
-Zenoh key **MUST** start with `upl/` or `upr/`.
-While the local UUri maps to `upl/`, the remote UUri maps to `upr/`.
-If UAuthority is defined in UUri, it **MUST** be put in the second part.
-Then the rest of the UUri **MUST** be put in the last part.
 
-To be more specific:
+* `up/`: Zenoh key **MUST** always start with `up/` and the scheme (`up://`) in the short UUri is always removed.
+* UAuthority: Although UAuthority does not exist in local UUri, it **MUST** be used in Zenoh key. We can use the UAuthority which is initialized in the constructor in this case.
+* `UE_ID`, `UE_VER_MAJOR`, and `URESOURCE_ID`: These should be kept in Zenoh Key and also separated by `/`.
+
+To be more specific, the format of Zenoh Key is:
+
+`up/[UAuthoriy]/[UE_ID]/[UE_VER_MAJOR]/[URESOURCE_ID]`
```

Comment:
This is 1:1 mapping of short form URI now so I would just state that you use the short form URI for zenoh key expressions.